ubuntu/github-repo

```diff
@@ -1 +1 @@
-https://github.com/tianon/docker-brew-ubuntu-core
+https://git.launchpad.net/cloud-images/+oci/ubuntu-base
```

Comment:
I went and ran the template generation scripts to make sure we didn't end up with anything silly like `https://git.launchpad.net/cloud-images/+oci/ubuntu-base/issues` from this and it all looks good :+1: